gdal/doc/source/programs/gdal2tiles.rst

```diff
@@ -87,10 +88,16 @@ can publish a picture without proper georeferencing too.
 
   .. versionadded:: 2.3
 
+.. option:: --tilesize=<TILESIZE>
+
+  Pixel size of the tiles.
+
+  .. versionadded:: 3.0.5
```

Comment:
Being a new feature, this will be:
```suggestion
  .. versionadded:: 3.1
```

Reply:
Changed it to 3.1 (Sorry I'm not aware of the Commit suggestion Feature)